examples/db-templates/mariadb-ephemeral-template.json

```diff
@@ -16,7 +16,8 @@
   },
   "message": "The following service(s) have been created in your project: ${DATABASE_SERVICE_NAME}.\n\n       Username: ${MYSQL_USER}\n       Password: ${MYSQL_PASSWORD}\n  Database Name: ${MYSQL_DATABASE}\n Connection URL: mysql://${DATABASE_SERVICE_NAME}:3306/\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/sclorg/mariadb-container/blob/master/10.1/README.md.",
   "labels": {
-    "template": "mariadb-persistent-template"
+    "template": "mariadb-persistent-template",
+    "app": "mariadb-persistent-template"
```

Comment:
we don't want "-template" in these labels.  compare what "oc new-app" produces for the label when run w/ the template.

Reply:
The `app` label should be parameterized so that when a DB template is used with another application and we want to associate it with that app we can.